src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -152,8 +152,8 @@ const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange,
         cellRenderer: ({ rowIndex }) => {
           const io = sortedData[rowIndex]
           const { name, optional, inputType } = io
-          const value = config[which][name] || ''
-          const error = errors[which][name]
+          const value = _.get([which, name], config) || ''
+          const error = _.get([which, name], errors)
```

Comment:
in cases where the snapshot reference isn't found, the call to validate() fails. Then, the `errors` object doesn't contain an `outputs` key. Without the lodash safety of `_.get`, this causes failures when clicking on the outputs for a workflow.